recipes/init.rb

```diff
@@ -54,3 +54,20 @@
     end
   end
 end
+
+include_recipe 'krb5' if hadoop_kerberos?
+
+krb5_principal "cdap/#{node['fqdn']}" do
```

Comment:
can we re-use ``default['cdap']['kerberos']['cdap_principal']`` ?  else, can we just get rid of all the ``default['cdap']['kerberos']`` stuff?